synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -428,13 +428,13 @@ StateBone_Context::StateBone_Context(CanvasView *canvas_view) :
 	Layer::Handle layer = get_canvas_interface()->get_selection_manager()->get_selected_layer();
 
 	if(Layer_SkeletonDeformation::Handle::cast_dynamic(layer)){
-		get_work_area()->set_type_mask(get_work_area()->get_type_mask() - (Duck::TYPE_TANGENT | Duck::TYPE_WIDTH));
+		get_work_area()->set_type_mask(get_work_area()->get_type_mask() - Duck::TYPE_TANGENT | (Duck::TYPE_WIDTH | Duck::TYPE_WIDTHPOINT_POSITION));
```

Comment:
I think it's better to use bit logic here, just to avoid getting unpredictable results in case when some flag is not enabled. 
I made an example showing this: https://godbolt.org/z/T494be75z

Reply:
Oh, as `-` was used everywhere, I thought there was an overload operator method.
Fixing it.

Reply:
Argh, it doesn't build without casting :P 
`get_work_area()->set_type_mask(Duck::Type(get_work_area()->get_type_mask() & ~(Duck::TYPE_TANGENT | Duck::TYPE_WIDTH | Duck::TYPE_WIDTHPOINT_POSITION)));`
or
`get_work_area()->set_type_mask(get_work_area()->get_type_mask() & Duck::Type(~(Duck::TYPE_TANGENT | Duck::TYPE_WIDTH | Duck::TYPE_WIDTHPOINT_POSITION)));`

Reply:
Oh it is overloaded: 
https://github.com/synfig/synfig/blob/310aaad378224b6db22cc0639e66e36dfc45368c/synfig-studio/src/gui/duck.h#L462-L465

Reply:
Ok.